frb_codegen/src/main.rs

```diff
@@ -26,3 +26,195 @@ fn main() -> anyhow::Result<()> {
     info!("Now go and use it :)");
     Ok(())
 }
+
+mod tests {
+    use std::fs;
+
+    use lazy_static::lazy_static;
+    use lib_flutter_rust_bridge_codegen::init_logger;
+
+    lazy_static! {
+        static ref LOGGER: () = init_logger(".", true).unwrap();
+    }
+
+    // VS Code runs in frb_codegen with "Run test" and flutter_rust_bridge with "Debug test" >_>
+    #[allow(dead_code)]
+    fn set_dir() {
+        if let Ok(metadata) = fs::metadata("frb_codegen") {
+            if metadata.is_dir() {
+                std::env::set_current_dir("frb_codegen").unwrap();
+            }
+        }
+    }
+
+    #[test]
+    fn pure_dart() {
+        // `cargo build` can fail because frb_example/pure_dart makes testing a build step! *gasp*
+        // If you want to get `cargo build` working again and investigate why pure_dart fails:
+        // 1. mv ../frb_example/pure_dart/rust/build.rs ../rb_example/pure_dart/rust/_build.rs
+        // 2. Run this test in the debugger - which is a copy of build.rs (but more complete)
+
+        assert!(cfg!(feature = "chrono"));
+        assert!(cfg!(feature = "uuid"));
+
+        use lib_flutter_rust_bridge_codegen::{
+            config_parse, frb_codegen, get_symbols_if_no_duplicates, RawOpts,
+        };
+        use std::process::Command;
+
+        set_dir();
+
+        /// Path of input Rust code
+        const RUST_INPUT: &str = "../frb_example/pure_dart/rust/src/api.rs";
+        /// Path of output generated Dart code
+        const DART_OUTPUT: &str = "../frb_example/pure_dart//dart/lib/bridge_generated.dart";
+
+        let _ = *LOGGER;
+
+        // Tell Cargo that if the input Rust code changes, to rerun this build script.
+        println!("cargo:rerun-if-changed={RUST_INPUT}");
```

Comment:
hmm this is useless I guess? since we are running a test, not in build.rs

Reply:
fixed